python/ray/tune/tune.py

```diff
@@ -389,8 +389,10 @@ def run(
             experiment regardless of the checkpoint_freq. Default is False.
             This has no effect when using the Functional Training API.
         verbose: 0, 1, 2, or 3. Verbosity mode.
-            0 = silent, 1 = only status updates, 2 = status and brief trial
-            results, 3 = status and detailed trial results. Defaults to 3.
+            0 = silent, 1 = default, 2 = verbose.
+            If ``RAY_AIR_NEW_OUTPUT=0``, uses the old verbosity settings:
```

Comment:
can we fully align this with the `env.rst` doc?

Reply:
Can you elaborate what you mean? The environment variables are documented in `ray-air/user-guides.rst`. The `tune/api/env.rst` are meant for tune-specific variables and is referenced by the section in the AIR user guide

Reply:
ah, in reviewing the files tab, it looked like your changes in env.rst could be aligned / replicated. maybe that's not the case. if so, please ignore